applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_pw_line_element.cpp

```diff
@@ -0,0 +1,128 @@
+
+ModelPart& CreateModelPartWithSolutionStepVariables(Model& rModel)
+{
+    auto& r_result = rModel.CreateModelPart("Main");
+    r_result.AddNodalSolutionStepVariable(WATER_PRESSURE);
+    r_result.AddNodalSolutionStepVariable(DT_WATER_PRESSURE);
+    r_result.AddNodalSolutionStepVariable(VOLUME_ACCELERATION);
+
+    return r_result;
+}
+
+TransientPwLineElement<2, 2> TransientPwLineElementWithPWDofs(const Properties::Pointer& rProperties,
+                                                              const Geometry<Node>::Pointer& rGeometry)
+{
+    auto result = TransientPwLineElement<2, 2>{
+        1, rGeometry, rProperties, {CalculationContribution::Permeability, CalculationContribution::Compressibility}};
+    for (auto& node : result.GetGeometry()) {
+        node.AddDof(WATER_PRESSURE);
+        node.AddDof(DT_WATER_PRESSURE);
+    }
+
+    return result;
+}
+
+TransientPwLineElement<2, 2> TransientPwLineElementWithPWDofs(Model& rModel, const Properties::Pointer& rProperties)
+{
+    auto& r_model_part = CreateModelPartWithSolutionStepVariables(rModel);
+
+    PointerVector<Node> nodes;
+    nodes.push_back(r_model_part.CreateNewNode(0, 0.0, 0.0, 0.0));
+    nodes.push_back(r_model_part.CreateNewNode(1, 1.0, 1.0, 0.0));
+    const auto p_geometry = std::make_shared<Line2D2<Node>>(nodes);
+    return TransientPwLineElementWithPWDofs(rProperties, p_geometry);
+}
+
+} // namespace
+
+namespace Kratos::Testing
+{
+using namespace Kratos;
+
+KRATOS_TEST_CASE_IN_SUITE(TransientPwLineElementReturnsTheExpectedLeftHandSideAndRightHandSide,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    const auto p_properties = std::make_shared<Properties>();
+    p_properties->SetValue(YOUNG_MODULUS, 1.000000e+07);
+    p_properties->SetValue(POISSON_RATIO, 0.000000e+00);
+    p_properties->SetValue(DENSITY_SOLID, 2.650000e+03);
+    p_properties->SetValue(DENSITY_WATER, 1.000000e+03);
+    p_properties->SetValue(POROSITY, 1.000000e-01);
+    p_properties->SetValue(BULK_MODULUS_SOLID, 1.000000e+12);
+    p_properties->SetValue(BULK_MODULUS_FLUID, 200.0); // small to get a significant value for the compressibility term
+    p_properties->SetValue(PERMEABILITY_XX, 9.084000e-06);
+    p_properties->SetValue(DYNAMIC_VISCOSITY, 1.0E-2);
+    p_properties->SetValue(BIOT_COEFFICIENT, 1.000000e+00);
+    p_properties->SetValue(RETENTION_LAW, "SaturatedLaw");
+    p_properties->SetValue(SATURATED_SATURATION, 1.000000e+00);
+    p_properties->SetValue(RESIDUAL_SATURATION, 0.000000e+00);
+    p_properties->SetValue(CROSS_AREA, 1.0);
+    p_properties->SetValue(IGNORE_UNDRAINED, false);
+
+    auto process_info                     = ProcessInfo{};
+    process_info[DT_PRESSURE_COEFFICIENT] = 1.5;
+
+    Model model;
+    auto  element = TransientPwLineElementWithPWDofs(model, p_properties);
+    element.GetGeometry()[0].FastGetSolutionStepValue(VOLUME_ACCELERATION) =
+        array_1d<double, 3>{0.0, -10.0, 0.0};
```

Comment:
This repeats below.

Reply:
I meant the 0.0, -10.0, 0.0 is repeated and could perhaps be named and reused. However that is a very very secondary and minor thing.